org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/binding/wizard/connection/ServerConnectionWizard.java

```diff
@@ -298,6 +298,7 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
       return false;
     } catch (InterruptedException e) {
       model.setUserOrgs(null);
+      Thread.currentThread().interrupt();
```

Comment:
Eclipse uses InterruptedException to notify of job cancellation. This is not truly a `Thred.interrupt`, so I think it is wrong (and possibly dangerous) to interrupt the current thread.

Reply:
Ok, I was not sure about this one. InterruptedException is normally specific to threads, didn't know Eclipse used them for jobs. So yes I should remove that and just ignore the issue